csharp/Bot/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using csharp;
+using Interfaces;
+using Octokit;
+using Platform.Exceptions;
+using Platform.IO;
+using Storage;
+using Storage.Local;
+using Storage.Remote.GitHub;
+using System;
+using System.Collections.Generic;
+
+namespace Bot
+{
+    class Program
+    {
+        private static void Main(string[] args)
+        {
+            using ConsoleCancellation cancellation = new ConsoleCancellation();
+            var username = ConsoleHelpers.GetOrReadArgument(0, "Username", args);
+            var token = ConsoleHelpers.GetOrReadArgument(1, "Token", args);
+            var appName = ConsoleHelpers.GetOrReadArgument(2, "App Name", args);
+            var databaseFileName = ConsoleHelpers.GetOrReadArgument(3, "Database file name", args);
+            var fileSetName = ConsoleHelpers.GetOrReadArgument(4, "File set name ", args);//For defoult Hello World: HelloWorldSet
+            var dbContext = new FileStorage(databaseFileName);
+            Console.WriteLine("Bot has been started.\nPress CTRL+C to close");

```

Comment:
https://stackoverflow.com/questions/3821784/whats-the-difference-between-n-and-r-n

Reply:
Use `Environment.NewLine` or one more `Console.WriteLine()`.